solutions/secure/README.md

```diff
@@ -0,0 +1,8 @@
+# IBM Elastic Search on IBM Cloud
+
+This architecture creates an IBM Elastic Search instance in IBM Cloud environment and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Elastic search instance on IBM Cloud
```

Comment:
what about the other things it supports?

Reply:
I have added CBR rules and autoscaling